cmd/tracing/daemon.go

```diff
@@ -20,20 +20,20 @@ import (
 	"sync/atomic"
 	"time"
 
-	"github.com/aws/aws-xray-daemon/daemon/bufferpool"
```

Comment:
Is this change necessary for moving to go.mod? Can we scope it down non-code changes for faster rollout?

Reply:
I have not changed any code.  I am just reorganizing the folders for best practices.  The only thing that has changed is the import directives.
https://github.com/golang-standards/project-layout#pkg